### ISSUE TITLE
Connection leak issue

### ISSUE THREAD

**Issue by vikas-tikoo-zefr** [domoinc/domo-java-sdk#3]
Getting the following warning log repeatedly:

```May 23, 2017 8:08:48 PM okhttp3.internal.platform.Platform log
WARNING: A connection to https://api.domo.com/ was leaked. Did you forget to close a response body? To see where this was allocated, set the OkHttpClient logger level to FINE: Logger.getLogger(OkHttpClient.class.getName()).setLevel(Level.FINE);```

**Comment by checketts** [comment]
What details were you able to see from setting the logger to `FINE`?

**Comment by checketts** [comment]
Fixed with 0.2.0 release
